src/chainparams.cpp

```diff
@@ -283,7 +283,6 @@ class CTestNetParams : public CChainParams
 
         consensus.vDeployments[Consensus::DEPLOYMENT_DAEDALUS].bit = 27;
         consensus.vDeployments[Consensus::DEPLOYMENT_DAEDALUS].start_block = 500;
-        consensus.vDeployments[Consensus::DEPLOYMENT_DAEDALUS].end_block = 5000;
```

Comment:
what's the default value of end_block, i forgot.

Reply:
It should default to zero, when it's not set.  

https://github.com/meritlabs/merit/blob/master/src/versionbits.cpp#L194

Reply:
The default is 0. If it's not 0, then end block check is activated.